disdat/hyperframe.py

```diff
@@ -526,7 +528,7 @@ def detect_local_fs_path(series):
     """
     output = []
     for s in series:
-        if not isinstance(s, str) and not isinstance(s, unicode):
+        if not isinstance(s, str) and not isinstance(s, str):
```

Comment:
Well, that's not right. 
```suggestion
        if not isinstance(s, str):
```

Reply:
Fixed